source/ldclint/visitors.d

```diff
@@ -26,7 +26,10 @@ import dmd.nspace;
 import dmd.init;
 import dmd.cond;
 
-import dmd.root.rootobject;
+static if (__VERSION__ >= 2108)
+    import dmd.rootobject;
+else
+    import dmd.root.rootobject;
```

Comment:
I have a bridge set of modules for these situations at `source/ldclint/dmd`. I can fix it later.